src/Raven.Client/Documents/Session/IAbstractDocumentQuery.cs

```diff
@@ -17,61 +17,66 @@ namespace Raven.Client.Documents.Session
     ///</summary>
     public interface IAbstractDocumentQuery<T>
     {
+        /// <summary>
+        ///     Get the queried index name.
+        /// </summary>
         string IndexName { get; }
         
+        /// <summary>
+        ///     Get the queried collection name.
+        /// </summary>
         string CollectionName { get; }
 
         /// <summary>
-        /// Gets the document convention from the query session
+        ///     Get the document conventions used for this query.
         /// </summary>
         DocumentConventions Conventions { get; }
 
         /// <summary>
-        /// Determines if it is a dynamic map-reduce query
+        ///     Determines if query is a dynamic map-reduce query.
         /// </summary>
         bool IsDynamicMapReduce { get; }
 
         /// <summary>
-        ///   Instruct the query to wait for non stale results.
-        ///   This shouldn't be used outside of unit tests unless you are well aware of the implications
+        ///     Instruct the query to wait for non-stale results.
+        ///     This shouldn't be used outside of unit tests unless you are well aware of the implications.
         /// </summary>
-        /// <param name = "waitTimeout">Maximum time to wait for index query results to become non-stale before exception is thrown. Default: 15 seconds.</param>
+        /// <param name="waitTimeout">Maximum time in seconds to wait for index query results to become non-stale before exception is thrown. Default: 15 seconds.</param>
         void WaitForNonStaleResults(TimeSpan? waitTimeout = null);
 
         /// <summary>
-        ///   Gets the fields for projection
+        ///     Gets field names of query result projection.
         /// </summary>
         /// <returns></returns>
         IEnumerable<string> GetProjectionFields();
 
         /// <summary>
-        /// Order the search results randomly
+        ///     Order the query results randomly.
         /// </summary>
         void RandomOrdering();
 
         /// <summary>
-        /// Order the search results randomly using the specified seed
-        /// this is useful if you want to have repeatable random queries
+        ///     Orders the query results randomly using the specified seed.
+        ///     Allows to repeat random query results.
         /// </summary>
         void RandomOrdering(string seed);
 
 #if FEATURE_CUSTOM_SORTING
         /// <summary>
-        /// Sort using custom sorter on the server
+        ///     Sort query results using server-side custom sorter.
+        ///     Requires custom sorting feature to be enabled.
         /// </summary>
+        /// <param name="descending">Changes order to descending.</param>
```

Comment:
Can we reference to https://ravendb.net/docs/article-page/6.0/csharp/client-api/session/querying/sort-query-results#custom-sorters ?

Reply:
Added